en/reference/annotations-mapping.rst

```diff
@@ -95,6 +99,7 @@ Full example:
     *   referenceable=true,
     *   translator="child",
     *   mixins={"mix:created", "mix:lastModified"}
+        childClasses={"SomeDocument", "AnotherDocument"}
```

Comment:
lets use App\Documents\Article or something. an example with a namespace, to make it clear that its not relative names or anything.